backend/.env.template

```diff
@@ -28,18 +28,18 @@ TRANSCRIPTION_MAX_FILE_SIZE=10485760
 MAX_IN_QUESTION=1
 
 # Feature flags
-USING_ACCESS_MANAGEMENT=
-USING_AZURE_MODELS=
+USING_ACCESS_MANAGEMENT=0
+USING_AZURE_MODELS=0
```

Comment:
These should probably be some variation of `False` instead, to be more descriptive.

Reply:
Done 😊